hexrdgui/physics_package_manager_dialog.py

```diff
@@ -142,7 +142,7 @@ def setup_form(self):
         self.ui.pinhole_density.setValue(physics.pinhole_density)
         if self.instrument_type == 'PXRDIP':
             self.ui.pinhole_thickness.setValue(70)
-            self.ui.pinhole_diameter.setValue(130)
+            self.ui.pinhole_diameter.setValue(300)
```

Comment:
Maybe these should be set by the physics package too?

Reply:
This was to account for the different default pinhole dimensions in PXRDIP and TARDIS. I think it's still editable in the physics package.